corehq/apps/appstore/templates/appstore/project_info.html

```diff
@@ -50,6 +50,8 @@
                 $('#agree-button').attr('data-form', new_form);
             });
             $('#agree-button').click(function() {
+                $('#agree-button').unbind()
+                                  .addClass('disabled');
```

Comment:
mostly habit - I tend to break up code that might be changed independently on separate lines for git history readability.  Although I agree the formatting is a bit weird here.